_studio/shared/umc/codec/jpeg_dec/src/jpegdec_base.cpp

```diff
@@ -110,6 +110,8 @@ void CJPEGDecoderBase::Reset(void)
   m_marker                 = JM_NONE;
 
   m_nblock                 = 0;
+
+  m_stream_in.Close();
```

Comment:
not sure for SetSource(), if pBuf = 0 we should set m_stream_in = 0 or  keep old value (in this case SetSource() will returns error)?

Reply:
If I were a designer of this class, I would call Close first inside Open. But for minimizing changes let's keep current implementation. those functions actually do almost nothing, so I don't think it would be a problem